tests/cases/employee_health_portal/case_employee_health_portal_account.yaml

```diff
@@ -316,4 +316,94 @@ TestEmployeeHealthPortalAccountHelp:
           Value: Employee_Health_User_Manual
         - Type: contain
           Var: HELP_PAGE_URL
-          Value: .pdf
\ No newline at end of file
+          Value: .pdf
+#CXTB-109 Employee Health Portal - Request an Appointment for User
+TestEmployeeHealthPortalRequestAnAppointmentForUser:
```

Comment:
This test  should not be part of the case_employee_health_portal_account.yaml file. This functionality is from the Request  Appointment section. Please create a new case yaml file for it

Reply:
Done